starport/services/networkbuilder/simulation.go

```diff
@@ -49,23 +46,18 @@ func (b *Builder) VerifyProposals(ctx context.Context, chainID string, proposals
 	}
 	defer os.RemoveAll(tmpHome)
 
-	// find out the app's name form url
-	u, err := url.Parse(chainInfo.URL)
-	if err != nil {
-		return false, err
-	}
-	importPath := path.Join(u.Host, u.Path)
-	path, err := gomodulepath.Parse(importPath)
+	appPath := filepath.Join(sourcePath, chainID)
+	path, err := gomodulepath.ParseFile(appPath)
 	if err != nil {
 		return false, err
 	}
 	app := chain.App{
-		ChainID:  chainID,
-		Name:     path.Root,
-		Version:  cosmosver.Stargate,
-		HomePath: tmpHome,
```

Comment:
Why removing it? We use a temporary directory because the simulation modify the ports in the config and we don't want to do this in the config in the official home dir

Reply:
of course, fixed by 50170ee.